jaffa-soa/src/test/org/jaffa/transaction/services/AllXmlLoadTest.java

```diff
@@ -0,0 +1,24 @@
+package org.jaffa.transaction.services;
```

Comment:
The folder structure is incorrect. This should be under src/test/java/org/jaffa/transaction/services